spec/integration/.kitchen.yml

```diff
@@ -0,0 +1,19 @@
+---
+driver:
+  name: docker
+  image: centos:6.7
+  use_sudo: true
```

Comment:
That's a known pain to me. Inside my infra I stick to `use_sudo: false` everywhere including CI. It just means that user which invokes `docker` is added to `docker` group, thus need no `sudo` to use docker. Unfortunately travis strongly needs `sudo` for docker scenario...